docs/custom-flows/magic-links.mdx

```diff
@@ -43,50 +43,40 @@ We take care of the boring stuff, like efficient polling, secure session managem
 - You need to create a Clerk Application in your [Clerk Dashboard](https://dashboard.clerk.com/). For more information, check out our [Set up your application](/docs/quickstarts/setup-clerk) guide.
 - You need to install the correct SDK for your application. You can find steps on how to do so through Clerk's [quickstart](/docs/quickstarts/overview) guides.
 
-## Configuration
+## Set up magic link authentication in Your Clerk application
 
-Magic link authentication can be configured through the Clerk Dashboard. Go to **User & Authentication > [Email, Phone, and Username](https://dashboard.clerk.com/last-active?path=user-authentication/email-phone-username)**. In the **Authentication factors** section of this page, choose **Email verification link** as the authentication factor.
+To enable magic link authentication, you need to configure your instance to allow magic links. You can do this through the Clerk Dashboard or by creating a custom flow using the Clerk API.
```

Comment:
I assume there's a separate PR for the "email link" rename, yeah? Just noting that this will likely generate a merge conflict, usually it's easiest to resolve those quickly before they add up and overlap 😅